deployment/ansible/group_vars/test

```diff
@@ -1 +1,4 @@
 ---
+docker_users:
```

Comment:
I think that this needs to be renamed to `jenkins` for the `raster-foundry-jenkins.yml` playbook to pick it up.